### ISSUE TITLE
Integration tests: Test authentication + authorisation

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2824]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2821
- Created at 2015-10-07 08:24:28 by [lslebodn](https://pagure.io/user/lslebodn)
- Assigned to nobody
---

This test requires pam_wrapper (su_wrapper). And it should prepare sample test for pam responder and maybe also sudo responder.

### Comments

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2015-10-07 08:29:29](https://pagure.io/SSSD/sssd/issue/2821#242495)**

Fields changed

cc:  => #2822

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2015-10-07 11:27:34](https://pagure.io/SSSD/sssd/issue/2821#242496)**

I would like to work on this ticket as part of making pam_wrapper useful

owner: somebody => jhrozek

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2015-10-07 11:40:21](https://pagure.io/SSSD/sssd/issue/2821#242497)**

Tests don't need BZ clone

rhbz:  => 0

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2015-10-08 23:26:20](https://pagure.io/SSSD/sssd/issue/2821#242498)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.13.3

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2015-12-10 16:38:09](https://pagure.io/SSSD/sssd/issue/2821#242499)**

Unfortunately we need to release 1.13.3 tarball soon and this ticket depends on pam_wrapper which is still under review..

milestone: SSSD 1.13.3 => SSSD 1.14 alpha

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2016-06-20 12:00:29](https://pagure.io/SSSD/sssd/issue/2821#242500)**

Fields changed

milestone: SSSD 1.14 alpha => SSSD Continuous integration

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2016-10-07 12:02:48](https://pagure.io/SSSD/sssd/issue/2821#242501)**

This is a task for the next milestone, we need to upstream the pam-wrapper tests.

milestone: SSSD Continuous integration => SSSD 1.15 Alpha

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2017-01-25 12:15:48](https://pagure.io/SSSD/sssd/issue/2821#242502)**

Fields changed

milestone: SSSD 1.15.0 => SSSD 1.15 Beta

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2017-02-24 15:06:52](https://pagure.io/SSSD/sssd/issue/2821#242503)**

**Metadata Update from @lslebodn**:
- Issue assigned to jhrozek
- Issue marked as blocked by: #2822
- Issue set to the milestone: SSSD 1.15.3

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2017-03-31 18:08:44](https://pagure.io/SSSD/sssd/issue/2821#434528)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch reset (from 0)
- Custom field review reset (from 0)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue **un**marked as depending on: #2822

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2017-03-31 18:11:00](https://pagure.io/SSSD/sssd/issue/2821#434537)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue set to the milestone: SSSD 1.15.4 (was: SSSD 1.15.3)

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2017-08-18 16:58:45](https://pagure.io/SSSD/sssd/issue/2821#458787)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue tagged with: cleanup-one-sixteen

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2017-08-23 17:21:16](https://pagure.io/SSSD/sssd/issue/2821#459971)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue **un**tagged with: cleanup-one-sixteen
- Issue set to the milestone: SSSD 1.16.0 (was: SSSD 1.15.4)

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2017-10-19 20:22:53](https://pagure.io/SSSD/sssd/issue/2821#473612)**

Since we are required to release a new upstream tarball no later than Friday Oct-20, I'm moving tickets that will not be closed by that date to the next milestone, 1.16.1

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2017-10-19 20:22:54](https://pagure.io/SSSD/sssd/issue/2821#473613)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue set to the milestone: SSSD 1.16.1 (was: SSSD 1.16.0)

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2017-12-15 20:01:11](https://pagure.io/SSSD/sssd/issue/2821#484940)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue tagged with: postpone-to-2-0

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2018-01-08 17:57:04](https://pagure.io/SSSD/sssd/issue/2821#487227)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue **un**tagged with: postpone-to-2-0
- Issue set to the milestone: SSSD 2.0 (was: SSSD 1.16.1)

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2018-03-20 13:53:39](https://pagure.io/SSSD/sssd/issue/2821#500566)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue priority set to: trivial (was: major)

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2018-08-13 11:12:00](https://pagure.io/SSSD/sssd/issue/2821#525891)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue set to the milestone: SSSD 2.1 (was: SSSD 2.0)

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2019-02-22 15:53:03](https://pagure.io/SSSD/sssd/issue/2821#555731)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue set to the milestone: SSSD 2.2 (was: SSSD 2.1)

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2019-06-13 23:24:52](https://pagure.io/SSSD/sssd/issue/2821#576128)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue set to the milestone: SSSD 2.3 (was: SSSD 2.2)

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2019-08-28 15:40:46](https://pagure.io/SSSD/sssd/issue/2821#593226)**

**Metadata Update from @jhrozek**:
- Assignee reset

---

**Comment from [lslebodn](https://pagure.io/user/lslebodn) at [2020-03-11 11:13:35](https://pagure.io/SSSD/sssd/issue/2821#632232)**

**Metadata Update from @thalman**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue tagged with: tests
